docs/fundamentals/accessibility.md

```diff
@@ -0,0 +1,378 @@
+
+> [!IMPORTANT]
+> Using the `AutomationProperties` attached properties may impact UI Test execution on Android. The `AutomationId`, `AutomationProperties.Name` and `AutomationProperties.HelpText` properties will both set the native `ContentDescription` property, with the `AutomationProperties.Name` and `AutomationProperties.HelpText` property values taking precedence over the `AutomationId` value (if both `AutomationProperties.Name` and `AutomationProperties.HelpText` are set, the values will be concatenated). This means that any tests looking for `AutomationId` will fail if `AutomationProperties.Name` or `AutomationProperties.HelpText` are also set on the element. In this scenario, UI Tests should be altered to look for the value of `AutomationProperties.Name` or `AutomationProperties.HelpText`, or a concatenation of both.
+
+Different screen readers read different accessibility values. For example, on Windows, Narrator will prioritize `AutomationProperties.Name`, `AutomationProperties.LabeledBy`, and then `AutomationProperties.HelpText`. On Android, TalkBack may combine the `AutomationProperties.Name` and `AutomationProperties.HelpText` values. Therefore, it's recommended that thorough accessibility testing is carried out on each platform to ensure an optimal experience.
+
+### ExcludedWithChildren
+
+The `AutomationProperties.ExcludeWithChildren` attached property, of type `bool?`, determines if an element and its children should be excluded from the accessibility tree. This enables scenarios such as displaying an `AbsoluteLayout` over another layout such as a `StackLayout`, with the `StackLayout` being excluded from the accessibility tree when it's not visible. It can be used from XAML as follows:
+
+```xaml
+<StackLayout AutomationProperties.ExcludedWithChildren="true">
+...
+</StackLayout>
+```
+
+Alternatively, it can be set in C# as follows:
+
+```csharp
+StackLayout stackLayout = new StackLayout();
+...
+AutomationProperties.SetExcludedWithChildren(stackLayout, true);
+```
+
+When this attached property is set, .NET MAUI sets the `AutomationProperties.IsInAccessibleTree` attached property to `false` on the specified element and its children.
+
+### IsInAccessibleTree
+
+> [!WARNING]
+> This attached property should typically remain unset. The majority of controls should be present in the accessibility tree, and the `AutomationProperties.ExcludedWithChildren` attached property can be set in scenarios where an element and its children need removing from the accessibility tree.
+
+The `AutomationProperties.IsInAccessibleTree` attached property, of type `bool?`, determines if the element is visible to screen readers. It must be set to `true` to use the other automation properties. This can be accomplished in XAML as follows:
```

Comment:
> It must be set to `true` to use the other automation properties.
I can't remember if this is necessarily true. @PureWeen do you?

At least with semanticProperties, when we set a description on something, it'll automatically switch isinacessibletree=true as well (i.e. images)

Reply:
AFAICR it's different per platform here.  On `Android` if you set the contentDesc of an element then `Android` will just set that element as being ImportantForAccessibility and read it out, but on iOS you have to set isInAccessibleTree to `true` 

So it's probably best to just indicate that you have to set it.